### DISCUSSION TITLE
page页面默认是64kb，如果超出64kb，某个字段值是1M以上这个字段如何存储

### DISCUSSION THREAD

**Discussion by easymanlo** [apache/doris#6743] [A - General / Q&A]
segment_v2存储引擎采用列式存储，一个datapage页面默认是64kb，如果某个字段超出64kb，1M以上这个字段如何存储？

**Comment by morningman** [comment]
page size can be larger than 64k

**Reply by easymanlo** [reply]
大于64KB但也有临界值的，例如是1M，总有一个临界值超过1M。例如位图类型的字段，一般都是几十M，这种情况，是不是位图存储得跨页了？还是通过其他方式解决
